data/stats/get_runway_stats.py

```diff
@@ -0,0 +1,37 @@
+""" Gather runway statistics for each 
+    airport from given airport runway data. 
+"""
+
+from dataclasses import dataclass, field
+from typing import TypedDict
+
+import pandas as pd
+
+
+class RunwayDict(TypedDict):
+    length_in_ft: int
+    length_in_m: int
+    name: str
+    surface: str
+
+
+@dataclass
+class RunwayStats:
+    runways: pd.DataFrame
+    _runways_stats_df: pd.DataFrame = field(init=False)
+
+    def __post_init__(self) -> None:
+        self._runways_stats_df = pd.DataFrame(self.runways.map(self.count_runways))
+        self._runways_stats_df["total_runway_length"] = pd.DataFrame(
+            self.runways.map(self.sum_runways_len)
```

Comment:
What does this return if not a DataFrame?

Reply:
It returns a DataFrame that's a single column of lists